app/services/mailer.js

```diff
@@ -0,0 +1,35 @@
+const nconf = require('nconf');
+const { createTransport } = require('nodemailer');
+const { defaultsDeep } = require('lodash');
+
+const logger = require('../utils/logger')('mailer');
+
+const transporter = createTransport({
+  service: 'Gmail',
+  secure: true,
+  auth: {
+    user: nconf.get('mailer:user'),
+    pass: nconf.get('mailer:pass'),
+  },
+  logger
+});
+
+const defaultEmailOptions = {
+  from: 'EventEmitterService@gmail.com',
```

Comment:
move to config

Reply:
[Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/20/commits/f5f1e52333bd5100b40935f350b7a08a372abf79#diff-a18e55398c3a9a4054717c2ccfa889fdR16)